### ISSUE TITLE
How about two stage training, should I add ghm loss in both rpn_head and two_stage_head ?

### ISSUE THREAD

**Issue by nemonameless** [libuyu/GHM_Detection#15]
or just rpn_head ?
and  should I use the same momentum ,mu and bins as the one stage detector?   And if I use cascade_mask_rcnn, how about these  hyperparameters?

Thanks a lot.

**Comment by libuyu** [comment]
It works on the classification branch of RPN in our early experiments. And the hyperparameters need a little finetuning.

However, although the proposals from RPN are improved, the final performance after the second stage has only a little improvement. So we just focus on the one-stage detector.

And this loss doesn't work on the two_stage_head, because two_stage detector uses a sampling strategy that avoids the imbalanced distribution of examples. Explanations in detail can be seen in this paper:  https://arxiv.org/abs/1708.02002.
